### ISSUE TITLE
Slow loss evolution

### ISSUE THREAD

**Issue by Vicktoria13** [Sisso16/VocAlign#1]
Hello

Thanks for the code !
I try to run it without changing anything and with batch size = 2 on cityscapes 

The loss is slowly going down is that expected ?
2.944
2.944
.
.
.
2.993

Ect.. is it expected ?Could you provide the weight or the logs ?

Also, you are usine à resnet 101 backbone. However it is Never used or mention in catseg paper in its version. So i am a little confused on where des it come from
Thanks !!

**Comment by Sisso16** [comment]
Hello,

Thanks for the interest in our work!

For the training loss you see, I would guess you didn't correctly load the checkpoint of the pre-trained CAT-Seg from which the adaption starts from. Those values indicate that you are adapting starting from random CAT-Seg weights.

I attach the screenshot of the log which shows the correct loading of the checkpoint and the log of the training in the first few epochs:

<img width="1549" height="165" alt="Image" src="https://github.com/user-attachments/assets/a209a2f2-aff2-400d-9c37-d3b124dbe952" />

<img width="1552" height="173" alt="Image" src="https://github.com/user-attachments/assets/bbbab4a4-9fed-488a-9fc4-3975729086f2" />

Concerning the use of the additional ResNet 101, it was used in the previous version of CAT-Seg (while we were working on this project) and it was then removed in the final published version of CAT-Seg. You can check this out in their official repo [CAT-Seg](https://github.com/cvlab-kaist/CAT-Seg), in the Introduction section of the README.
